packages/twenty-server/src/engine/core-modules/ai/constants/ai-models.const.ts

```diff
@@ -0,0 +1,71 @@
+export enum ModelProvider {
+  OPENAI = 'openai',
+  ANTHROPIC = 'anthropic',
+}
+
+export interface AIModelConfig {
+  modelId: string;
+  displayName: string;
+  provider: ModelProvider;
+  inputCostPer1kTokensInCents: number;
+  outputCostPer1kTokensInCents: number;
+  isActive: boolean;
```

Comment:
Not a big fan of isActive as we don't really know what it means (past workflows already created with that model will work or not?). Maybe remove it if you're not sure how to do something clear